frontend/lib/pages/data-driven-onboarding.tsx

```diff
@@ -191,11 +191,10 @@ const buildingIntroCard: ActionCardPropsCreator = (data): ActionCardProps => {
         Your building was built in {data.yearBuilt} or earlier.
       </>,
       !hasHpdRegistration && (isBuildingClassBorC(data.buildingClass) 
-        ? <><span className="has-text-danger has-text-weight-semibold">No registration found.</span> Your landlord may be breaking the law! </> 
-        : <span className="has-text-danger has-text-weight-semibold">No registration found.</span>),
-      !hasHpdRegistration && (isBuildingClassBorC(data.buildingClass)
-        ? <>It looks like this building may require registration with HPD. Landlords who don't properly register their properties incur fines and also cannot bring tenants to court for nonpayment of rent. You can find more information on <OutboundLink href="https://www1.nyc.gov/site/hpd/services-and-information/register-your-property.page" target="_blank">HPD's Property Management page</OutboundLink>.</>
-        : <>It doesn't seem like this property is required to register with HPD. You can learn about the City's registration requirements on <OutboundLink href="https://www1.nyc.gov/site/hpd/services-and-information/register-your-property.page" target="_blank">HPD's Property Management page</OutboundLink>.</>)
+        ? <><span className="jf-registration-warning"><span className="has-text-danger has-text-weight-semibold">No registration found.</span> Your landlord may be breaking the law!</span>
+          <>It looks like this building may require registration with HPD. Landlords who don't properly register their properties incur fines and also cannot bring tenants to court for nonpayment of rent. You can find more information on <OutboundLink href="https://www1.nyc.gov/site/hpd/services-and-information/register-your-property.page" target="_blank">HPD's Property Management page</OutboundLink>.</></> 
```

Comment:
The spacing here is definitely a bit cramped, but of course codeclimate was yelling at me for making this function over 25 lines so I had to condense somehow...

Reply:
Oy I really wish we could tell codeclimate to ignore the 25 lines thing, at least for JSX that's clearly content/markup rather than business logic.

Note though that we do have the ability to tell codeclimate that specific instances of its warnings are actually invalid.  If you are logged into their site and click the "details" link next to the CodeClimate check at the bottom of this page, you should see a little drop-down next to each warning that lets you resolve issues, e.g.:

> ![image](https://user-images.githubusercontent.com/124687/74533406-81d3f480-4eff-11ea-8d4d-c6e26f5b2d4e.png)

You can learn about the meaning of all those statuses on [CC's issues documentation](https://docs.codeclimate.com/docs/issues#section-issue-statuses).  I think that we can safely mark any of CC's warnings about JSX functions being too long as "Invalid"!